z2/src/docgen.rs

```diff
@@ -144,6 +163,56 @@ fn insert_key(val: &mut serde_yaml::Value, components: &Vec<String>, idx: usize,
     }
 }
 
+#[derive(Debug, Clone, PartialEq, Hash, Eq)]
+pub enum ApiMethod {
+    JsonRpc { name: String },
+    Rest { uri: String },
```

Comment:
Do we need this? I'd avoid the enum if we don't have any rest endpoints. Easy to add in the future, but annoying to keep it around if we never use it.

Reply:
We have the health check endpoint, and anything to do with websockets (which I will admit is going to be hard to include in this anyway)